File: learn.python/NumberPattern.py
''' 
You can define the Value of 'rows' when you need to
Print the Pattern Only for a Fixed number of Rows
'''
# rows = 6

# if you want user to enter a number, below line does it
rows = int(input("\nEnter the number of rows : "))

# outer loop
for i in range(rows):
    # nested loop
    for j in range(i):
        # display number
        print(i, end=' ')
    # new line after each row
    print('')

File: learn.java/NumberPatterns/NumberPattern.java
// package NumberPatterns;

import java.util.*;

class NumberPattern {   
    public static void main(String[] args) {

        // Initializing requeired Variables
        int i, j, k = 1;
        int rows;
        
        // New Scanner Object for getting User Input
        Scanner input = new Scanner(System.in);
        System.out.println("\nEnter the number of Rows : ");
        rows = input.nextInt();

        // Loop to Print the Number Pattern
        System.out.println("\n");
        for (i = 1; i <= rows; i++) {
            for (j = 1; j< i + 1; j++) {
                System.out.print(k++ + " ");
            }
            System.out.println();
        }
    }
}
